quantum/qmk_settings.c

```diff
@@ -27,16 +27,18 @@ static void auto_shift_timeout_apply(void) {
 }
 
 #ifdef MOUSEKEY_ENABLE
-static void mousekey_apply(void) {
-    mk_delay = QS.mousekey_delay / 10;
-    mk_interval = QS.mousekey_interval;
-    mk_max_speed = QS.mousekey_max_speed;
-    mk_time_to_max = QS.mousekey_time_to_max;
-    mk_wheel_delay = QS.mousekey_wheel_delay / 10;
-    mk_wheel_interval    = QS.mousekey_wheel_interval;
-    mk_wheel_max_speed   = QS.mousekey_wheel_max_speed;
-    mk_wheel_time_to_max = QS.mousekey_wheel_time_to_max;
-}
+#    ifndef MK_3_SPEED
+    static void mousekey_apply(void) {
+        mk_delay = QS.mousekey_delay / 10;
+        mk_interval = QS.mousekey_interval;
+        mk_max_speed = QS.mousekey_max_speed;
+        mk_time_to_max = QS.mousekey_time_to_max;
+        mk_wheel_delay = QS.mousekey_wheel_delay / 10;
+        mk_wheel_interval    = QS.mousekey_wheel_interval;
+        mk_wheel_max_speed   = QS.mousekey_wheel_max_speed;
+        mk_wheel_time_to_max = QS.mousekey_wheel_time_to_max;
+    }
+#    endif
```

Comment:
Change to `#if defined(MOUSEKEY_ENABLE) && !defined(MK_3_SPEED)`, remove whitespace change